libappstream-glib/as-content-rating.c

```diff
@@ -1134,7 +1134,7 @@ static const struct {
 	{ "drugs-narcotics",	OARS_1_0, 0, 12, 14, 17 },
 	{ "drugs-tobacco",	OARS_1_0, 0, 10, 13, 13 },
 	{ "sex-nudity",		OARS_1_0, 0, 12, 14, 14 },
-	{ "sex-themes",		OARS_1_0, 0, 13, 14, 15 },
+	{ "sex-themes",		OARS_1_0, 0, 13, 14, 18 },
```

Comment:
The 15 there was deliberate after much debate with a child physiologist (aka my wife). We'd much rather lower the other one to match, rather than raise both.

Reply:
Done!